### DISCUSSION TITLE
Maintainance

### DISCUSSION THREAD

**Discussion by abhishekr700** [codereverser/casparser#59] [General]
Hi @codereverser , will you be keeping this updated? I'm thinking of building sort of a personal wealth tracker (own use, nothing fancy) and this package can help me in automating the import of mutual fund txns. 
So, just wondering, have you commited to keeping this updated over time ?

**Comment by codereverser** [comment]
Yep, that's the plan at least. 

 I use this module frequently for personal use, so I try to keep this updated whenever possible.

**Reply by abhishekr700** [reply]
That's so good to know. If you need help in the maintainance part, I would love to contribute. But that would require some going over the codebase where I'll need your help. 
Thanks again for creating such a useful module !!!
